pointcloud/src/main/scala/geotrellis/pointcloud/spark/triangulation/BoundaryDelaunay.scala

```diff
@@ -209,12 +209,16 @@ case class BoundaryDelaunay (dt: DelaunayTriangulation, boundingExtent: Extent)
           // r.showBoundingLoop(r.getFlip(r.getNext(opp)))
           case None =>
             //println("     --- DID NOT FIND TRIANGLE")
+            val tri = copyConvertTriangle(e)
+            navigator.join(opp, tri)
+
             if (circumcircleLeavesExtent(boundingExtent)(e)) {
               //println("         Triangle circle leaves extent")
-              val tri = copyConvertTriangle(e)
+      //        val tri = copyConvertTriangle(e)
+
               //print("         ")
               //navigator.showLoop(tri)
-              navigator.join(opp, tri)
+      //        navigator.join(opp, tri)
```

Comment:
The salient change in this PR is to comment lines 217 and 221 and move them to lines 212 and 213.